atomic_reactor/odcs_util.py

```diff
@@ -81,6 +87,10 @@ def start_compose(self, source_type, source, packages=None, sigkeys=None, arches
         if arches is not None:
             body['arches'] = arches
 
+        if multilib_arches:
+            body['multilib_arches'] = multilib_arches
+            body['multilib_method'] = sorted(set(multilib_method or MULTILIB_METHOD_DEFAULT))
```

Comment:
Trying to make a sorted list with unique elements?

Do we care about element uniqueness, or list ordering? I feel like we ought to just accept what the user gave us.

Reply:
The sorted is to make testing more sane, I think.

@mlangsdorf, I think if we sort just on the tests it should be enough:
```python
# tests/test_odcs_util.py ~ line 124
assert sorted(body_json.get('multilib_method')) == sorted(expected_method)
```
Just accepting what was given seems reasonable to me.

Reply:
There are a couple of places in `test_resolve_composes` that this should be done too. That way we can also remove the sorting for both `multilib_arches` and `multilib_method` in `pre_resolve_composes` plugin.

Reply:
Fixed.